stacks/api-stack.js

```diff
@@ -1,12 +1,61 @@
-import { Api } from '@serverless-stack/resources'
+import {
+  Api,
+  Bucket,
+  Table
+} from '@serverless-stack/resources'
+
+import { getConfig } from './config.js'
 
 /**
  * @param {import('@serverless-stack/resources').StackContext} properties
  */
 export function ApiStack({ stack }) {
+  // @ts-ignore "prod" | "dev" | "staging" only allowed for stage
+  const stackConfig = getConfig(stack.stage)
+
+  /**
+   * This table takes a stored CAR and makes an entry in the store table
+   * to associate the uploaders DID with a payload CID.
+   * You can also optionally indicate an application DID, origin and size
+   */
+   const storeTable = new Table(stack, 'store_table', {
+    fields: {
+      uploaderDID: 'string',
+      payloadCID: 'string',
+      applicationDID: 'string',
+      origin: 'string',
+      size: 'number',
+      proof: 'string',
+      uploadedAt: 'string',
+    },
+    primaryIndex: { partitionKey: 'uploaderDID', sortKey: 'payloadCID' },
+    ...stackConfig.tableConfig,
+  })
+
+  /**
+   * This bucket is the car store used within the `store/*` invocations.
+   * Force name of bucket to be "car-park-prod-0" in prod.
+   */
+  const ingestBucketCdkConfig =
+    stack.stage === 'prod' ? { cdk: { bucket: { bucketName: 'car-park-prod-0' } } } : {}
+  
+  const carStore = new Bucket(stack, 'carStore', {
```

Comment:
in this file we have
- kebab-case
- camelCase
- snake_case

for resource ids. plz can we not.